public/vendor/nova/app.css

```diff
@@ -0,0 +1,3 @@
.md\:col-span-6{grid-column:span 6/span 6}.md\:col-span-3{grid-column:span 3/span 3}.md\:col-span-8{grid-column:span 8/span 8}.md\:col-span-9{grid-column:span 9/span 9}.md\:mt-0{margin-top:0}[dir=ltr] .md\:ml-3{margin-left:.75rem}[dir=rtl] .md\:ml-3{margin-right:.75rem}[dir=ltr] .md\:ml-2{margin-left:.5rem}[dir=ltr] .md\:mr-2,[dir=rtl] .md\:ml-2{margin-right:.5rem}[dir=rtl] .md\:mr-2{margin-left:.5rem}.md\:mt-6{margin-top:1.5rem}.md\:inline-block{display:inline-block}.md\:flex{display:flex}.md\:grid{display:grid}.md\:hidden{display:none}.md\:w-1\/5{width:20%}.md\:w-4\/5{width:80%}.md\:w-3\/5{width:60%}.md\:w-1\/4{width:25%}.md\:w-3\/4{width:75%}.md\:w-\[20rem\]{width:20rem}.md\:w-1\/3{width:33.333333%}.md\:flex-shrink-0{flex-shrink:0}.md\:grid-cols-12{grid-template-columns:repeat(12,minmax(0,1fr))}.md\:grid-cols-2{grid-template-columns:repeat(2,minmax(0,1fr))}.md\:flex-row{flex-direction:row}.md\:items-center{align-items:center}.md\:justify-end{justify-content:flex-end}.md\:space-y-0>:not([hidden])~:not([hidden]){--tw-space-y-reverse:0;margin-bottom:calc(0px*var(--tw-space-y-reverse));margin-top:calc(0px*(1 - var(--tw-space-y-reverse)))}.md\:space-x-20>:not([hidden])~:not([hidden]){--tw-space-x-reverse:0}[dir=ltr] .md\:space-x-20>:not([hidden])~:not([hidden]){margin-left:calc(5rem*(1 - var(--tw-space-x-reverse)));margin-right:calc(5rem*var(--tw-space-x-reverse))}[dir=rtl] .md\:space-x-20>:not([hidden])~:not([hidden]){margin-left:calc(5rem*var(--tw-space-x-reverse));margin-right:calc(5rem*(1 - var(--tw-space-x-reverse)))}.md\:space-y-4>:not([hidden])~:not([hidden]){--tw-space-y-reverse:0;margin-bottom:calc(1rem*var(--tw-space-y-reverse));margin-top:calc(1rem*(1 - var(--tw-space-y-reverse)))}.md\:border-b-0{border-bottom-width:0}.md\:px-8{padding-left:2rem;padding-right:2rem}.md\:py-5{padding-bottom:1.25rem;padding-top:1.25rem}.md\:py-0{padding-bottom:0;padding-top:0}.md\:py-3{padding-bottom:.75rem;padding-top:.75rem}.md\:py-8{padding-bottom:2rem;padding-top:2rem}.md\:px-12{padding-left:3rem;padding-right:3rem}.md\:px-2{padding-left:.5rem;padding-right:.5rem}.md\:px-3{padding-left:.75rem;padding-right:.75rem}.md\:px-0{padding-left:0;padding-right:0}.md\:py-6{padding-bottom:1.5rem;padding-top:1.5rem}.md\:pt-2{padding-top:.5rem}[dir=ltr] .md\:pr-3{padding-right:.75rem}[dir=rtl] .md\:pr-3{padding-left:.75rem}.md\:text-2xl{font-size:1.5rem;line-height:2rem}.md\:text-xl{font-size:1.25rem;line-height:1.75rem}.md\:text-sm{font-size:.875rem;line-height:1.25rem}.md\:text-\[4rem\]{font-size:4rem}.md\:font-normal{font-weight:400}}@media (min-width:1024px){.lg\:absolute{position:absolute}.lg\:top-\[56px\]{top:56px}.lg\:bottom-auto{bottom:auto}[dir=ltr] .lg\:ml-60{margin-left:15rem}[dir=rtl] .lg\:ml-60{margin-right:15rem}.lg\:block{display:block}.lg\:hidden{display:none}.lg\:w-60{width:15rem}.lg\:max-w-lg{max-width:32rem}.lg\:break-words{overflow-wrap:break-word}.lg\:px-6{padding-left:1.5rem;padding-right:1.5rem}.lg\:px-12{padding-left:3rem;padding-right:3rem}}
```

Comment:
Can we ignore all these "vendor" files? (`public/vendor`, `lang/vendor`, ...)

Reply:
Those files are added by running the `nova::install` command, see `https://nova.laravel.com/docs/4.0/installation.html

I took the approach of only requiring `composer install` to be run, but it can be changed of course.

@bertramakers what do you prefer?